isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/nutzungsvorgaben/inhalt.adoc

```diff
@@ -56,34 +162,58 @@ public class OAuth2WebClientConfiguration {
     @Bean("webclient")
     WebClient webClient(ClientRegistrationRepository clientRegistrationRepository,
         OAuth2AuthorizedClientService authorizedClientService) {
-        ServletOAuth2AuthorizedClientExchangeFilterFunction oauth2 =
+        ServletOAuth2AuthorizedClientExchangeFilterFunction oauth =
             new ServletOAuth2AuthorizedClientExchangeFilterFunction(
                 new AuthorizedClientServiceOAuth2AuthorizedClientManager(clientRegistrationRepository,
                     authorizedClientService));
         oauth.setDefaultClientRegistrationId("registrationId");
-        return WebClient.builder().apply(oauth2.oauth2Configuration()).build();
+        return WebClient.builder().filter(oauth).build();
     }
 
 }
 ----
 
-[[konfigurationsparameter]]
-=== Konfigurationsparameter
+[[manual-authentication]]
+=== Manuelle Authentifizierung eines Clients mit dem Authentifizierungsmanager
 
-Für die korrekte Anbindung an den `Identity Provider` ist die Angabe einiger Konfigurationsparameter notwendig. Das Präfix sämtlicher Konfigurationsparameter `spring.security.oauth2` wird zur Vereinfachung in der Liste unten weggelassen.
+Die manuelle Authentifizierung eines Client erfolgt über den Authentifizierungsmanager, der entsprechende Methoden zur Verwendung mit und ohne zuvor konfigurierte `ClientRegistrations` bereitstellt.
 
+1. Authentifizierung per `registrationId`.
+Die Auswahl des Flows (_Client-Credentials_ oder _Resource-Owner-Password-Credentials_) wird dabei über den `authorization-grant-type` der vollständig konfigurierten `ClientRegistration` getroffen (siehe <<listing-client-registrations>>).
+Diese Methode eignet sich in Anwendungen wenn die Zugangsdaten für Technische Nutzer zur Konfiguration bekannt sind.
+2. Authentifizierung per `clientId`, `clientSecret` und `issuerLocation`.
+Diese Methode nutzt den _Client-Credentials_ Flow und kann verwendet werden wenn die zugangsdaten (`clientId` & `clientSecret`) erst zur Laufzeit bekannt sind.
+3. Authentifizierung per `username`, `password`, `registrationId` und (optional) `bhknz`.
+Diese Methode nutzt den _Resource-Owner-Password-Credentials_ Flow und setzt eine entsprechende `ClientRegistration` mit `authorization-grant-type=password` voraus.
```

Comment:
With ISY-147 there are no wording `manual authentification` any more and the supported authentification method amount and signatures of the `AuthentificationManager` have be changed.